src/test/java/com/epam/jamp2/service/CalculationCommandExecutionServiceIT.java

```diff
@@ -34,7 +34,12 @@
     public void testCalcWithDiffCcy() throws CommandFormatException, IOException {
         CalculationCommand cmd = CalculationCommand.parseFromString(ADD_DIDD_CCY);
         Value result = calculationCommandExecutionService.calculate(cmd);
-        validateCalculatedValue(result, FxRateMockData.CNY, 691.6, 5.0);
+
+        Assert.assertNotNull(result);
+        Assert.assertThat(result.getCurrencyCode().get(), is(FxRateMockData.CNY));
+        Assert.assertThat(result.getValue(),
+                is(closeTo(new BigDecimal(691.6), new BigDecimal(5.0))));
```

Comment:
Even though it's an integration test, probably it's not a good idea to rely on current rates - they may change significantly over time and your closeTo() matcher will eventually fail. I would simply check that result is calculated and not NULL here, that would be enough. While in other unit tests I would check if calculations are implemented correctly.